test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Export-Counter.Tests.ps1

```diff
@@ -0,0 +1,216 @@
+<############################################################################################
+ # File: Export-Counter.Tests.ps1
+ # Provides Pester tests for the Export-Counter cmdlet.
+ ############################################################################################>
+Describe "Tests for Export-Counter cmdlet" -Tags "CI" {
+
+    BeforeAll {
+        $cmdletName = "Export-Counter"
+
+        . "$PSScriptRoot/CounterTestsCommon.ps1"
```

Comment:
The helper functions are used by multiple test scripts. I have, as we discussed, renamed the file to CounterTestHelperFunctions.ps1, and made additional changes in that file.